File: chatwoot/csharp/src/OSEG.ChatwootExamples/DeleteAUserExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class DeleteAUserExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey = new Dictionary<string, string> {["platformAppApiKey"] = "PLATFORM_APP_API_KEY"};

        try
        {
            new UsersApi(config).DeleteAUser(
                id: null
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling UsersApi#DeleteAUser: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/typescript-axios/src/DeleteAUserExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
  apiKey: "PLATFORM_APP_API_KEY",
});

new api.UsersApi(configuration).deleteAUser(
  0, // id
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling UsersApi#deleteAUser:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
